onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -526,7 +529,7 @@ def aten_avg_pool1d(
 
     raise NotImplementedError()
 
-
+@torch_op("aten::baddbmm")
 def aten_baddbmm(
     self: TensorType,
```

Comment:
```suggestion
    self: TFloat,
```

Does TFloat seem to be an appropriate type for this? We want a type that is the broadest supported type by the ONNX ops used in the function. You can refer to tensor_typing.py to find or create a type.

Reply:
In aten_mul, why don't we care about uint8?
https://github.com/microsoft/onnx-script/blob/78ea55b888de88bfdadce7c3f6f3f83fa1404c7f/onnxscript/function_libs/torch_aten/ops/core.py#L3440
https://github.com/onnx/onnx/blob/main/docs/Changelog.md#Mul-14

Reply:
We probably should. It must be a mistake. Since we are only testing with float inputs rn I expect we miss corner cases here and there. #417

Reply:
> Does TFloat seem to be an appropriate type for this? We want a type that is the broadest supported type by the ONNX ops used in the function. You can refer to tensor_typing.py to find or create a type.

I think you mean the intersection of supported types across all used ONNX ops in that function?

Reply:
You said it better. That’s the idea. I will put it in #388